### ISSUE TITLE
Best Practices Industry Leaders

### ISSUE THREAD

**Issue by jaimiebutton** [bcgov/github-optimization#64]
Do we have a list of industry leaders or governments we can say we used to inform best practices? Even if it's a small list or organizations, it may be helpful to point to them.

**Context: Slide 26 & 27**

**Comment by jlangy** [comment]
Yes, we can say that we looked at https://github.com/18F (technology and design consultancy for the U.S. Government), https://github.com/alphagov (UKgov digital service), https://github.com/canada-ca (Government of Canada)
